src/configure.rs

```diff
@@ -461,7 +467,8 @@ async fn configure(ctx: &Context, param: &mut LoginParam) -> Result<()> {
     }
 
     // the trailing underscore is correct
```

Comment:
Btw what is this comment about?

Reply:
Should be removed, it's about an older version of the code where `"configured_"` was passed as one of the arguments for the next function call